src/components/filters/MeasureValueFilter/tests/MeasureValueFilter.spec.tsx

```diff
@@ -0,0 +1,230 @@
+// (C) 2019 GoodData Corporation
+import * as React from "react";
+import { mount } from "enzyme";
+import noop = require("lodash/noop");
+
+import MVFDropdownFragment from "./fragments/MeasureValueFilter";
+import { DropdownAfmWrapper, IDropdownProps } from "../DropdownAfmWrapper";
+import * as Operators from "../../../../constants/measureValueFilterOperators";
+import { withIntl } from "../../../visualizations/utils/intlUtils";
+import { measureValueFilter } from "../../../../helpers/model/measureValueFilters";
+
+const renderComponent = (props?: Partial<IDropdownProps>) => {
```

Comment:
What if this function returned the `MVFDropdownFragment` instance directly? That way you wouldn't be able to interact with the rendered component in any other way than using the fragment, which I believe is desirable.

Reply:
Great idea, looks way better now :-)